File: code/coding_pratice_Interview/prime_num/prime_num.py
def prime_num(n):
    flag = 0
    for i in range(2, n):
        if(n % i == 0):
            flag = 1
    
    if(flag):
        print("It's Not a Prime Number")
    else:
        print("It's a Prime Number")

        


prime_num(7)

File: code/coding_pratice_Interview/prime_num/prime_num.js
const prime_num = (n) => {
    let flag = 0;
    for(let i = 2; i < n; i++){
        if(n % i == 0){
            flag = 1
        }
    }

    if(flag){
        console.log("The Given number is a prime number");
    }
    else{
        console.log("The Given number is not a Prime  number");    
    }
}

// prime_num(9)